src/Reflector/ClassReflector.php

```diff
@@ -32,11 +36,15 @@ public function __construct(SourceLocator $sourceLocator)
      */
     public static function buildDefaultReflector() : self
     {
-        return new self(new AggregateSourceLocator([
-            new PhpInternalSourceLocator(),
-            new EvaledCodeSourceLocator(),
-            new AutoloadSourceLocator(),
-        ]));
+        $locator = new PhpParserLocator(
```

Comment:
At this point, I think that we should drop the `PhpParserLocator` completely - it is an internal aspect that should not be exposed via DI